docs/source/for_matlab_users.rst

```diff
@@ -0,0 +1,14 @@
+pyttb for MATLAB users
+======================
+
+Already familiar with MATLAB Tensor Toolbox? To assist transitions from the
+Tensor Toolbox for MATLAB to pyttb, this guide documents some key differences.
```

Comment:
It might be good to mention that the tutorials are similar to the MATLAB ones if they want a hands on approach. I think that's already implied but might not be a bad idea to be explicit.